src/spring-cloud/azext_spring_cloud/_params.py

```diff
@@ -148,7 +148,9 @@ def prepare_logs_argument(c):
         with self.argument_context(scope) as c:
             c.argument(
                 'artifact_path', options_list=[
-                    '--artifact-path', '--jar-path', '-p'], help='If provided, deploy pre-built artifact (jar or netcore zip), otherwise deploy current folder as tar.')
+                    '--artifact-path', '--jar-path'], help='If provided, deploy pre-built artifact (jar or netcore zip), otherwise need to assign source_path.')
```

Comment:
keep the original abbreviated `-p`?

Reply:
For the message, maybe just `Deploy the specified pre-built artifact (jar or netcore zip).`

Reply:
I am not sure because there are two ways to provide path right now. I will confirm this with yuchen